core/spec/models/spree/reimbursement_spec.rb

```diff
@@ -118,7 +118,7 @@
           subject
         }.to change { Spree::Refund.count }.by(1)
         return_item.reload
-        expect(return_item.included_tax_total).to be < 0
+        expect(return_item.included_tax_total).to be > 0
```

Comment:
Reimbursements are positive; the negative included tax adjustment would be (I think) for a tax refund reimbursement. Since now there is no tax refunds anymore as the price is right from the start, the refund doesn't need to be reimbursed.